core/templates/dev/head/domain/story_viewer/ReadOnlyStoryNodeObjectFactory.ts

```diff
@@ -0,0 +1,92 @@
+// Copyright 2018 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Factory for storing frontend story node domain objects in the
+ * story viewer.
+ */
+
+require('pages/story-editor-page/story-editor-page.controller.ts');
```

Comment:
This is added because of the `NODE_ID_PREFIX`?

Reply:
Yeah, but the constant is not used, removed it.